projects/npm-tools/packages/npm-scripts/src/utils/getWorkspaces.js

```diff
@@ -10,8 +10,6 @@ const expandGlobs = require('./expandGlobs');
 const findRoot = require('./findRoot');
 const log = require('./log');
 
-const IGNORE_GLOBS = ['node_modules/**'];
```

Comment:
Mmm, I think I've seen some `node_modules` directories inside projects. We shouldn't have them because we are supposed to dedupe deps and hoist all of them, but in theory it is possible that we may need them.

Also, I believe `bin` commands need these folders to exist.

Reply:
Apparently you recover the globs in the second commit, so it's all OK :+1: